.github/workflows/develop.yml

```
@@ -0,0 +1,27 @@
+name: Develop jobs
```

Yes, this a bit pedantic, but the "jobs" suffix seems quite redundant... Everything in a workflow will be a job, so there's no need to have that in the name, I think?

```suggestion
name: Develop
```